statsmodels/stats/kmo.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Sep 10 20:21:04 2016
+
+KMO Test Function:
+
+- This test evaluates sampling adequacy for exploratory Factor Analysis
+
+KMO Test function has one input:
+
+- The Dataset Correlation Matrix
+
+It Outputs the test result, and the results per variable
+
+
+@authors: Rui Sarmento
+          Vera Costa
+"""
+
+import numpy as np
+import math as math
+import collections
+
+#KMO Test
+def kmo(dataset_corr):
+    
+    r"""

```

Comment:
This needs a summary descrotion line.
Some additional explanation what kmo is would also be helpful to users